lib/module.js

```diff
@@ -169,6 +169,9 @@ Module._findPath = function(request, paths) {
     }
 
     if (filename) {
+      if (request === '.' && i > 0) {
+        console.error(`(node) warning: require('.') resolved to ${filename}`);
+      }
```

Comment:
Any advice on how to use `util.deprecate` in this case? It seems `util.deprecate` is tuned for deprecating functions. In this case, I'd like to print a single warning if this condition is met.